setup.cfg

```diff
@@ -0,0 +1,8 @@
+[flake8]
+# Black takes care of line formatting so flake8 should not mess with them
+extend-ignore = E203, E501
+
+[isort]
+profile = black
+known_third_party = google,invoke,pydantic,pytest
+known_first_party = firedantic
```

Comment:
I don't think firedantic should be considered first-party and it's neither used within this repo anywhere, so could likely just get rid of this whole line.
```suggestion
```